src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -2323,13 +2345,23 @@ private async Task<T> Execute<T>(string message, Func<Task<T>> func)
                 catch (Exception ex)
                 {
                     line.Write($" Failed. {ex.GetType().Name}: {ex.Message}", ConsoleColor.Red);
-                    cts.Cancel();
+
```

Comment:
I've been thinking about this for a long time. This is OK for now, but in the future rather than hacking our way around this we should consider creating a test for role assignments to avoid this. If we don't have an issue, we should open one.